### PR TITLE
update basic-features.md for v8.5

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/19140
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -22,253 +22,253 @@ You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?u
...
+| [DDL algorithm assertions](/sql-statements/sql-statement-alter-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| Multi-schema change: add columns | Y | Y | Y | Y | Y | E | E | E | E | E |
+| [Change column type](/sql-statements/sql-statement-modify-column.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Temporary tables](/temporary-tables.md) | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
+| Concurrent DDL statements | Y | Y | Y | Y | Y | N | N | N | N | N |
+| [Acceleration of `ADD INDEX` and `CREATE INDEX`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) | Y | Y | Y | Y | Y | N | N | N | N | N |
+| [Metadata lock](/metadata-lock.md) | Y | Y | Y | Y | Y | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) | Y | Y | Y | Y | Y | N | N | N | N | N |
+| [Pause](/sql-statements/sql-statement-admin-pause-ddl.md)/[Resume](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | Y | N | N | N | N | N | N | N |
+| [TiDB Accelerated Table Creation](/accelerated-table-creation.md) | E | E | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:142]:
```suggestion
| [TiDB Accelerated Table Creation](/accelerated-table-creation.md) | Y | E | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -22,253 +22,253 @@ You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?u
...
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge-experimental) | E | E | E | E | E | E | E | E | E | E |
+| [Use buckets to improve scan concurrency](/tune-region-performance.md#use-bucket-to-increase-concurrency) | E | E | E | E | E | E | N | N | N | N |
+| [Invisible indexes](/sql-statements/sql-statement-create-index.md#invisible-index) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Composite `PRIMARY KEY`](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`CHECK` constraints](/constraints.md#check) | Y | Y | Y | N | N | N | N | N | N | N |
+| [Unique indexes](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Clustered index on integer `PRIMARY KEY`](/clustered-indexes.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Clustered index on composite or non-integer key](/clustered-indexes.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued-indexes) | Y | Y | Y | Y | N | N | N | N | N | N |
+| [Foreign key](/constraints.md#foreign-key) | E | E | E | E | N | N | N | N | N | N |
```
Comment (lilin90) [inline basic-features.md:67]:
```suggestion
| [Foreign key](/constraints.md#foreign-key) | Y | E | E | E | N | N | N | N | N | N |
```

File: basic-features.md
Comment (qiancai) [inline basic-features.md:1]: preview: https://pr.pingcap-docsite-preview.pages.dev/tidb/dev/basic-features

**Review by Oreoxmt — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: basic-features.md [hunk 3]
```diff
@@ -22,253 +22,253 @@ You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?u
...
+| [Selecting TiDB nodes for the Distributed eXecution Framework (DXF) tasks](/system-variables.md#tidb_service_scope-new-in-v740) | Y | Y | N | N | N | N | N | N | N |
+| PD Follower Proxy (controlled by [`tidb_enable_tso_follower_proxy`](/system-variables.md#tidb_enable_tso_follower_proxy-new-in-v530)) | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) (controlled by [`pd_enable_follower_handle_region`](/system-variables.md#pd_enable_follower_handle_region-new-in-v760)) | Y | E | N | N | N | N | N | N | N | N |
+| [PD microservices](/pd-microservices.md) | E | E | N | N | N | N | N | N | N | N |
+| [TiDB Distributed eXecution Framework (DXF)](/tidb-distributed-execution-framework.md) | Y | Y | Y | E | N | N | N | N | N | N |
+| [Global Sort](/tidb-global-sort.md) | Y | Y | E | N | N | N | N | N | N | N |
+| [TiProxy](/tiproxy/tiproxy-overview.md) | Y | Y | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:272]:
```suggestion
| [schema cache](/schema-cache.md) | Y | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -22,254 +22,254 @@ You can try out TiDB features on [TiDB Playground](https://play.tidbcloud.com/?u
...
+| [Schema cache](/schema-cache.md) | Y | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:272]:
```suggestion
| [Schema cache](/schema-cache.md) | Y | N | N | N | N | N | N | N | N | N |

```

**Pull request merged**